guides/using-json-rest-requests.md

```diff
@@ -0,0 +1,135 @@
+# Using Raw JSON REST Requests
```

Comment:
Let's try to match the format in https://github.com/opensearch-project/opensearch-py/blob/main/guides/json.md?

Reply:
@dblock I don't think this repo has a sample directory like the Python client does. Should I create a sample dir in this .NET client repo & put it there? Or did you mean you want me to put the sample somewhere else as in another repo?